apps/game_backend/lib/game_backend/users.ex

```diff
@@ -75,8 +74,11 @@ defmodule GameBackend.Users do
       )
 
     daily_quest_subquery =
-      from(dq in DailyQuest,
-        where: dq.inserted_at > ^start_of_date and dq.inserted_at < ^end_of_date and is_nil(dq.completed_at),
+      from(user_quest in UserQuest,
+        where:
+          user_quest.inserted_at > ^start_of_date and user_quest.inserted_at < ^end_of_date and
```

Comment:
Shouldn't we also be filtering by UserQuest.Quest.type == "daily" here? (through a join, obviously)

Reply:
right!